example/dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:22.04
+
+RUN apt update
+RUN apt install git python3 python3-pip -y
+RUN git clone https://github.com/ntu-eclab-ca-ta/PECO /root/peco
+
+RUN mkdir -p /root/project
+WORKDIR /root/project
+COPY judge.yaml /root/project/judge.yaml
+
+WORKDIR /root/peco
+RUN git checkout docker
```

Comment:
Because this is a merge from "docker" to "main", maybe the `checkout` is not needed.

Reply:
Thank you. The new commit has fixed it.

Reply:
I think we should add a tag to a commit which always works for the example.

Reply:
Sounds great. It seams like an undertaken released version for user.

Reply:
Test passed on my side. I think it's ok for merge.